### ISSUE TITLE
Reimplement custom log loss as a tf.keras object

### ISSUE THREAD

**Issue by tallamjr** [tallamjr/astronet#41]
When attempting to run evaluation and predictions on a loaded model that contains the custom loss, several errors occur.

It seems that from https://github.com/keras-team/keras/issues/5916 
https://github.com/keras-team/keras/issues/5916#issuecomment-457624404 one needs to point to the `custom_object`. Currently the implementation is not in this form.
